### DISCUSSION TITLE
3D - Strecke messen

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#156] [Q&A]
Von den  Waldaufsehern und dem betrieblichen Forstpersonal (extern) haben wir den dringenden Wunsch erhalten, dass der Azimut beim 3D -  Strecke messen so wie im "alten Atlas" dargestellt wird (siehe Screenshot).
Das würde die Kollegen und Kolleginnen bei Trassierungsarbeiten (Seilkran u. dgl.) sehr helfen.

alt:
![3dlänge_waldaufseher](https://github.com/gis-eni/webgis-forum/assets/100088983/15e2a393-8d88-4ebd-9723-96a00ceabfd5)

neu:

![3dlänge_wälaufseher_neu](https://github.com/gis-eni/webgis-forum/assets/100088983/8f5721d1-d51e-4114-92e3-23201957794b)

**Comment by gis-eni** [comment]
Kann man natürlich bei den 3D Werten hinschreiben. Grundsätzlich läuft das Azimut aber eh auch beim Zeichnen mit:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/7b6f27e6-5d38-4cf2-9f68-26425bfe8814)

Also gibt es erst mal einen Workaround dafür.

**Reply by faulgurke** [reply]
Geht aber nur, solange sie nicht auf "3D Länge ermitteln ..." klicken. Dann ist der Azimut wieder weg.
Vielen Dank!

**Comment by gis-eni** [comment]
Darum nenn ich es auch "Workaround"

**Comment by gis-eni** [comment]
Ein Lösung würde so aussehen:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/2026f622-409f-4512-aaf6-4563a62b28c3)

Die kommt dann im nächsten Build

**Reply by gis-eni** [reply]
Und noch eine Anmerkung dazu:
Wichtig ist das Koordinatensystem, mit dem die 3D Daten abgeholt wird. Das wird in der ``etc/measzre/3d.xml`` definiert, Attribute ``srs``:

``` xml
<xml>

<heightabovedatum srs="31256"
                  type="ims" 
                  server="..."
                  service="..."
                  rastertheme="..."
                  expression="{0:.00}"
                  />

</xml>
```

In diesem Koordinatensystem wird das Azimut auch berechnet. Wenn man zB GK verwendet, wird das Ergebnis auch plausibel sein. Verwendet man hier Web-Mercator, kommen ziemlich sicher (für alle Werte) keine vernüftigen Ergebnisse heraus.
